src/theme/dojo/index.ts

```diff
@@ -26,7 +26,7 @@ import * as listItem from './list-item.m.css';
 import * as loadingIndicator from './loading-indicator.m.css';
 import * as list from './list.m.css';
 import * as menuItem from './menu-item.m.css';
-import * as multiSelectTypeahead from './multi-select-typeahead.m.css';
+import * as multiSelectTypeahead from './chip-typeahead.m.css';
```

Comment:
Import this as `chipTypeahead` for consistency?

Reply:
Good catch!